### ISSUE TITLE
Error with variadics is misleading

### ISSUE THREAD

**Issue by kavigupta** [rahularya50/scheme_interpreter_personal#79]
![image](https://user-images.githubusercontent.com/3214932/55273548-2e769a80-528a-11e9-8ef1-52e3b5ea49bb.png)

The staff interpreter has the following error

```
scm> . y
Traceback (most recent call last):
  0	(variadic y)
Error: Cannot evaluate variadic symbol
```

**Comment by rahularya50** [comment]
Duplicate of #75 ?

**Comment by kavigupta** [comment]
Possibly, let's see if it gets resolved by #75

**Comment by kavigupta** [comment]
Resolved by #83
